### PR TITLE
support min function.

### PR DESCRIPTION
**Pull request by JaredTan95**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [x] New feature provided
- [ ] Improve performance

- Related issues
support Minimum aggregation and fix some unit test case.
___
### Bug fix
- Bug description.

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.

### CODE REVIEW

**Comment by wu-sheng** [comment]
First, could you share why adding this func?

**Comment by JaredTan95** [comment]
> First, could you share why adding this func?

simplely want to aggregation some metrics as follows:

```shell
service_latency_min = from(Service.latency).min();
endpoint_latency_min = from(Endpoint.latency).min();
```

**Comment by wu-sheng** [comment]
Interesting, but make sense in some cases.

**Comment by JaredTan95** [comment]
Yeah, and we also got `endpoint_latency_max` metrics by `Max` function, but found there is no `Min` function.

File: oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/metrics/MinLongMetrics.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+    protected static final String VALUE = "value";
+
+    @Getter @Setter @Column(columnName = VALUE, isValue = true) private long value;
+
+    @Entrance
+    public final void combine(@SourceFrom long count) {
+        if (this.value == 0) {
+            this.value = count;
+        }
```
Comment (kezhenxu94) [inline oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/metrics/MinLongMetrics.java]: An obvious bug here, try the test codes:

```java
        MinLongMetricsImpl impl = new MinLongMetricsImpl();
        impl.combine(10);
        impl.combine(0);
        impl.combine(10000);
```

I expect `impl.getValue()` to be `0`, but it's `10000`, please recheck @JaredTan95

Comment (kezhenxu94) [reply oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/metrics/MinLongMetrics.java]: Making `value` default to `Long.MAX_VALUE` is my first intuition

Comment (kezhenxu94) [reply oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/metrics/MinLongMetrics.java]: > An obvious bug here, try the test codes:
> 
> ```java
>         MinLongMetricsImpl impl = new MinLongMetricsImpl();
>         impl.combine(10);
>         impl.combine(0);
>         impl.combine(10000);
> ```
> 
> I expect `impl.getValue()` to be `0`, but it's `10000`, please recheck @JaredTan95

I suggest putting this into unit test as well

Comment (JaredTan95) [reply oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/metrics/MinLongMetrics.java]: Thanks to your experienced guidance, I will update it.

**Comment by wu-sheng** [comment]
@JaredTan95 Please take care of #3752

**Pull request merged**